recipe/meta.yaml

```diff
@@ -48,9 +43,6 @@ test:
     - pytest
     - pytest-xdist
     - mpmath
-{% if ver_major_minor == "1.3" %}
-    - blas * netlib  # [ppc64le]
```

Comment:
Could this be the source of the issues? Seems like a blas issue on ppc64le

Reply:
Ahhhhh. I’ll add that back with a 1.4 version constraint.